app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/backend.ts

```diff
@@ -283,6 +291,17 @@ export interface Asset<Type extends AssetType = AssetType> extends BaseAsset {
 /** The type returned from the "create directory" endpoint. */
 export interface Directory extends Asset<AssetType.directory> {}
 
+// =================
+// === Constants ===
+// =================
+
+export const ASSET_TYPE_NAME: Record<AssetType, string> = {
+    [AssetType.project]: 'project',
+    [AssetType.directory]: 'folder',
```

Comment:
why name change? could we keep it consistent and use `directory` here also?

Reply:
I think for all user facing messages we have been using "folder" instead of "directory"... but definitely possible to change if desired

Reply:
*to clarify: The Figma uses "Folders", so that's what I've been using for any user facing text